draft-ietf-quic-transport.md

```diff
@@ -865,11 +861,10 @@ longer if the peer chooses to not send STREAMS_BLOCKED frames.
 
 # Connections {#connections}
 
-A QUIC connection is a single conversation between two QUIC endpoints.  QUIC's
-connection establishment combines version negotiation with the cryptographic
-and transport handshakes to reduce connection establishment latency, as
-described in {{handshake}}.  Once established, a connection may migrate to a
-different IP or port at either endpoint as
+QUIC's connection establishment combines version negotiation with the
+cryptographic and transport handshakes to reduce connection establishment
+latency, as described in {{handshake}}.  Once established, a connection
+may migrate to a different IP or port at either endpoint as
```

Comment:
We only specify client-side migration.

Reply:
This is reflowed verbiage: I only made changes related to the "connection is a conversation" bit.

Reply:
@marten-seemann, I think this is an artifact of the effort to specify the migration primitives generically, even if only clients can migrate freely in v1.  Given the SPA mechanism, this is literally true as written, but more constrained that one might infer from this text.  Feel free to open a separate editorial issue for this, but I don't think we should make @dtikhonov fix it in his unrelated PR.